contracts/contracts/strategies/ThreePoolStrategy.sol

```diff
@@ -271,4 +303,15 @@ contract ThreePoolStrategy is InitializableAbstractStrategy {
         pToken.safeApprove(crvGaugeAddress, 0);
         pToken.safeApprove(crvGaugeAddress, uint256(-1));
     }
+
+    /**
+     * @dev Get the index of the coin in 3pool
+     */
+    function _getPoolCoinIndex(address _asset) internal view returns (int128) {
+        ICurvePool threePool = ICurvePool(platformAddress);
+        for (int128 i = 0; i < 3; i++) {
+            if (threePool.coins(uint256(i)) == _asset) return i;
```

Comment:
Wonder how much extra gas this is going to cost us. We'll be making an external function call six times every time we go through all three coins to do something. I guess with just USDT/USDC this could either be three calls or five.